src/Cli/dotnet/commands/dotnet-restore/RestoreCommandParser.cs

```diff
@@ -3,25 +3,38 @@
 
 using System.Collections.Generic;
 using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
 using System.Linq;
 using Microsoft.DotNet.Tools;
+using Microsoft.DotNet.Tools.Restore;
 using LocalizableStrings = Microsoft.DotNet.Tools.Restore.LocalizableStrings;
 
 namespace Microsoft.DotNet.Cli
 {
     internal static class RestoreCommandParser
     {
+        public static readonly string DocsLink = "https://aka.ms/dotnet-restore";
+
         public static readonly Argument<IEnumerable<string>> SlnOrProjectArgument = new Argument<IEnumerable<string>>(CommonLocalizableStrings.SolutionOrProjectArgumentName)
         {
             Description = CommonLocalizableStrings.SolutionOrProjectArgumentDescription,
             Arity = ArgumentArity.ZeroOrMore
         };
 
+        public static readonly Option<IEnumerable<string>> SourceOption = new ForwardedOption<IEnumerable<string>>(
+            new string[] { "-s", "--source" },
+            LocalizableStrings.CmdSourceOptionDescription)
+        {
+            ArgumentHelpName = LocalizableStrings.CmdSourceOption
+        }.ForwardAsSingle(o => $"-property:RestoreSources={string.Join("%3B", o)}")
```

Comment:
Is the Uri "%3B" required over the string ";"?  Should there be a comment that %3B is a semicolon?

Reply:
Added comment